tests/test_json/test_json_3.json

```diff
@@ -0,0 +1,29 @@
+{
+  "inputs": {
+    "cases": "input_cases.csv",
```

Comment:
Ok, so the original plan was to have `inputs` generated automatically by the job runner.

If we've got to manually tag specific inputs, we may need to revisit this.

Reply:
There are occasions when you want to do something _between_ the job runner extract and the matching, so we at least need an option to be able to manually specify inputs.